src/ontology/mondo-ingest-odk.yaml

```diff
@@ -33,6 +33,8 @@ components:
       source: https://data.bioontology.org/ontologies/ICD10CM/submissions/23/download?apikey=8b5b7825-538d-40e0-9e9e-5ab9274a9aeb
     - filename: icd10who.owl
       source: https://github.com/monarch-initiative/icd10who/releases/latest/download/icd10who.ttl
+    - filename: icd11.owl
+      source: https://icd11files.blob.core.windows.net/tmp/whofic-2023-04-08.owl.gz
```

Comment:
## Question: What format?
@matentzn is the format of the file in Manchester syntax?
<details><summary>Snippet</summary>
<p>

```
Prefix(:=<http://id.who.int/icd/schema/>)
Prefix(owl:=<http://www.w3.org/2002/07/owl#>)
Prefix(rdf:=<http://www.w3.org/1999/02/22-rdf-syntax-ns#>)
Prefix(xml:=<http://www.w3.org/XML/1998/namespace>)
Prefix(xsd:=<http://www.w3.org/2001/XMLSchema#>)
Prefix(rdfs:=<http://www.w3.org/2000/01/rdf-schema#>)


Ontology(<http://who.int/icd>
Annotation(:releaseDate "2023-04-09")
Annotation(:releaseId "beta"@en)
Annotation(rdfs:comment "The WHO Family of International Classifications (WHO-FIC) comprises classifications that have been endorsed by the WHO to describe various aspects of health and the health system in a consistent manner. The purpose of the Family is to assist the development of reliable statistical systems at local, national and international levels, with the aim of improving health status and health care. The three WHO-FIC Reference Classifications are: The International Classification of Diseases and Health Related Problems (ICD), the International Classification of Functioning, Disability and Health (ICF), and the
International Classification of Health Intervention (ICHI).

This document is a simplified OWL 2 rendering of the WHO-FIC Foundation Component that includes the ICD-11, ICHI, and ICF. This is work in progress, and it is being released for experimentation purposes only.

For more information, please contact icd@who.int."@en)
Annotation(owl:versionInfo "2023-04-09")

Declaration(Class(<http://id.who.int/icd/entity/1000004774>))
Declaration(Class(<http://id.who.int/icd/entity/1000034337>))
Declaration(Class(<http://id.who.int/icd/entity/100006598>))
Declaration(Class(<http://id.who.int/icd/entity/1000069301>))
Declaration(Class(<http://id.who.int/icd/entity/1000093173>))
Declaration(Class(<http://id.who.int/icd/entity/1000099724>))
```

</p>
</details>

Reply:
This is called functional syntax